### ISSUE TITLE
AudioStream in Animator preventing senes working or being called on Android 4.0.2.rc1 template

### ISSUE THREAD

**Issue by GrimmBros** [godotengine/godot#77967]
### Godot version

V4.0.2

### System information

Windows 10 /android 

### Issue description

Target SDK, using 32.0.0

If i have a scene that uses an Audiostream node in the animator it prevents the scene from being loaded by android devices, scence do work fine on PC still and on web exports, if i remove the Audiostream from the animation the scene can be loaded, 

Its fine to call AudioStreams and play them via code, its only when they are used through the animator that it completly prevents a scene from loading or Preloading

### Steps to reproduce

need to set up you Android Exports in order to reproduce

basicly take this file export to android debug and it will crash, remove the audiostream and it will work

### Minimal reproduction project

[Android Audio Crash.zip](https://github.com/godotengine/godot/files/11679600/Android.Audio.Crash.zip)

**Comment by AThousandShips** [comment]
> Android 4.0.2.rc1 template

Specifically then? Not 4.0.2 or later?

**Comment by GrimmBros** [comment]
i havent tried on later versions where can i get a later template?

**Comment by AThousandShips** [comment]
[Here](https://godotengine.org/article/maintenance-release-godot-4-0-3/), keep an eye on the website

**Comment by GrimmBros** [comment]
I'll give it an update some time tonight and test if it fixes the issue.

Will send an update after.

**Comment by GrimmBros** [comment]
I updated to 4.0.3, and it looks to be an error with Audiostream files made by godot 4.0.2
if i run the files i made befor, that are updated now to use 4.0.3 export templates (including remaking the android folder). They will still have the same issue and not load sences that contain the Audiostream files that have animation keyframes (only for android, they work on web and PC). however if i delet the Audiosteam, then make a new one in that same project and key the animations again it will load.

I've only tested loading these Scence as the main, still need to test if they can be called in, as i originaly fond the issue when move bettween scenes, and still havent corrected enough of the base project with new audiosteams in the animations to check it will work. 

I'll keep testing it out in a few more senarios and send another update over the next 12 hours.
